manager/controllers/utils.go

```diff
@@ -465,6 +466,14 @@ func buildQuery(key string, val string) string {
 	switch key {
 	case "sap_sids":
 		cmp = "::jsonb @> ?::jsonb"
+	case "sap_system":
+		// use presence of other sap fields to imply sap_system
+		isSap, _ := strconv.ParseBool(val)
+		not := "not "
+		if isSap {
+			not = ""
+		}
+		return fmt.Sprintf(`%sh.system_profile::jsonb ? 'sap_version'`, not)
```

Comment:
I would not change anything here in this function and in `ApplyTagsFilter`, except adding `"sap_version": true`, to `validFilters` map, then it will work fine.

When migration with inventory data will be fully completed later on, we would make the `"sap_system"` obsolete by removing it from `validFilters`.